opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntityList.js

```diff
@@ -392,7 +392,11 @@ const StixCoreRelationshipCreationFromEntityList = ({
                       </ListItemIcon>
                       <ListItemText
                         primary={node.name}
-                        secondary={defaultDescription(node)}
+                        secondary={
+                          <span style={{ whiteSpace: 'nowrap' }}>
+                            {defaultDescription(node)}
+                          </span>
+                        }
```

Comment:
The name and description are still overflowing:
![image](https://github.com/user-attachments/assets/60cba28a-97be-4a7f-a372-918f4c7a7135)

You can do this with CSS easily 😃 
```suggestion
                        secondary={defaultDescription(node)}
                        sx={{
                          '.MuiListItemText-primary, .MuiListItemText-secondary': {
                            overflowX: 'hidden',
                            textOverflow: 'ellipsis',
                            whiteSpace: 'nowrap',
                            marginRight: '20px',
                          },
                        }}
```
![image](https://github.com/user-attachments/assets/51791d82-2178-413f-9f71-03634a61cddc)